### PR TITLE
Removed outdated comments

### PR DESCRIPTION
**Pull request by bsuryadevara**
- Updated `run.py` file outdated comments for all the examples.

### CODE REVIEW

File: examples/abp_pcap_detection/run.py [hunk 1]
```diff
@@ -136,16 +137,20 @@ def run_pipeline(
             filter_null=False,
         ))
 
-    # Add a deserialize stage
+    # Add a deserialize stage.
+    # This stage deserializes source data into Dataframes.
```
Comment (dagardner-nv) [inline examples/abp_pcap_detection/run.py:141]: I think this is no longer true, the source stage emits MessageMeta objects, and the DeserializeStage simply chunks them into smaller MultiMessage's according to the config's `pipeline_batch_size`

File: examples/abp_pcap_detection/run.py [hunk 2]
```diff
@@ -154,28 +159,35 @@ def run_pipeline(
...
     pipeline.add_stage(AddClassificationsStage(config, labels=["probs"]))
 
-    # Add a monitor stage
+    # Add a monitor stage.
+    # This stage logs the metrics (msg/sec) from the above stage.
     pipeline.add_stage(MonitorStage(config, description="Add classification rate", unit="add-class"))
 
-    # Convert the probabilities to serialized JSON strings using the custom serialization stage
+    # Add a serialize stage.
+    # This stage include & exclude columns from messages.
```
Comment (dagardner-nv) [inline examples/abp_pcap_detection/run.py:175]:
```suggestion
    # This stage includes & excludes columns from messages.
```

File: examples/gnn_fraud_detection_pipeline/run.py [hunk 3]
```diff
@@ -120,39 +120,44 @@ def run_pipeline(
...
+    # In this stage, messages were loaded from a file.
     pipeline.set_source(FileSourceStage(
         config,
         filename=input_file,
         filter_null=False,
     ))
 
-    # Add a deserialize stage
+    # Add a deserialize stage.
+    # This stage deserializes source data into Dataframes.
```
Comment (dagardner-nv) [inline examples/gnn_fraud_detection_pipeline/run.py:135]: Same

**Review by mdemoret-nv — APPROVED** [review]

**Pull request merged**